tools/frr.in

```diff
@@ -337,7 +357,7 @@ stop_prio()
 	fi
 
 	wanted_prio=$1
-	daemon_list=${daemon:-$DAEMONS}
+	daemon_list=$(echo ${daemon:-$DAEMONS}| xargs -n1 | sort -g | xargs)
```

Comment:
Why is the list sorted here?  I thought zebra intentionally is at the beginning.